main.go

```diff
@@ -83,16 +86,17 @@ func main() {
 		Namespace:          namespace,
 	})
 	if err != nil {
-		setupLog.Error(err, "unable to start manager")
+		setupLog.Error(err, "Unable to start manager")
```

Comment:
Let's leave the message letter casing the way it was. These messages were generated by operator SDK and we might need to generate more in the future. So let's stick to the operator SDK message casing.

This comment should apply all casing changes no just this specific one